docs/RELEASE_NOTES.md

```diff
@@ -67,6 +68,9 @@ implementors of the `GenericCustomActionBuilder` interface. [#3](https://github.
 for secondary resources (e.g. assets and prices). [#3](https://github.com/commercetools/commercetools-sync-java/issues/3)
 - **Commons** - New Custom Type Id is now validated against being empty/null. [#3](https://github.com/commercetools/commercetools-sync-java/issues/3)
 
+**Enhancements** (1)
+- **Build Tools** - Bumped commercetools-jvm-sdk to version 1.29.0. [#262](https://github.com/commercetools/commercetools-sync-java/issues/262)
```

Comment:
maybe you can link the sdk-version to its release notes: http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/meta/ReleaseNotes.html#v1_29_0

Reply:
d83dd14